elm-src/index.elm

```diff
@@ -0,0 +1,489 @@
+module Main exposing (..)
+
+import NixOptions exposing (..)
+import NixJSON exposing (Gson, gnull, gsonDecoder, nixFromGson)
+import Paginate exposing (..)
+import Html exposing (node, Attribute, Html, a, text, em, table, tr, th, td, tbody, thead, input, ul, li, button, h1, h2, div, pre, p, hr)
+import Html.Attributes exposing (placeholder, src, disabled, href, class, autofocus, value, id)
+import Html.Events exposing (onInput, onClick)
+import Http
+import Json.Encode exposing (encode, Value, null)
+import Dict exposing (Dict, empty, foldr, filter)
+import List
```

Comment:
No need for the default imports (as listed here: http://package.elm-lang.org/packages/elm-lang/core/latest)